apps/condo/domains/acquiring/schema/MultiPayment.test.js

```diff
@@ -209,6 +211,23 @@ describe('MultiPayment', () => {
             })
         })
     })
+    describe('Validation tests', () => {
+        test('Should have correct dv field (=== 1)', async () => {
+            const admin = await makeLoggedInAdminClient()
+            const { payments, acquiringIntegration, client } = await makePayerAndPayments()
+            await expectToThrowValidationFailureError(async () => {
+                await createTestMultiPayment(admin, payments, client.user, acquiringIntegration, {
+                    dv: 2,
+                })
+            }, 'unknownDataVersion')
+            const [multiPayment] = await createTestMultiPayment(admin, payments, client.user, acquiringIntegration)
+            await expectToThrowValidationFailureError(async () => {
+                await updateTestMultiPayment(admin, multiPayment.id, {
+                    dv: 2,
+                })
+            })
+        })
+    })
     describe('real-life cases', () => {
         // TODO (savelevMatthew) write tests
```

Comment:
New TODO format should include Jira task number

Reply:
resolved 👍